### DISCUSSION TITLE
Has the yatai web ui slowed down since 0.10.x ?

### DISCUSSION THREAD

**Discussion by jiyer2016** [bentoml/BentoML#1335] [General] [answered]
I upgraded to 0.10.1 today - and found the yatai-web ui significantly slower than the norm. I timed a few page loads - and it seemed like it took ~60 seconds. A downgrade to 0.9.2 seemed to bring page load times to normal.

I haven't explored if this is a network issue on my end - but it seemed like it wasnt because the downgrade effectively fixed the issue. 

If this is really an issue - Should we open a bug report for this ?

**Comment by yubozhao** [comment, marked as answer]
@jiyer2016 This is interesting.  I don't think the few changes between `0.9.2` and `0.10.1` should significantly slow down the load time.

I will do some load tests on my end.  Can you tell me roughly how many bentos and deployments you have? Are you using labels, if you do roughly how many labels per resource?

I remembered from our previous conversations in Github, you are using s3 as repo storage and postgres for storing metadata.  Is that still the case?

Let's open an issue for this.  It has much higher visibility than discussion.

**Reply by jiyer2016** [reply]
I have 153 Bento's (under Repository) - None of them have "labels" attached. The Deployments page shows ZERO active deployments probably because I don't deploy via the CLI ?

And Yes, I use S3 (Minio) and PostgreSQL.

**Reply by jiyer2016** [reply]
Migrated to  https://github.com/bentoml/BentoML/issues/1339

**Comment by jiyer2016** [comment]
Migrating to https://github.com/bentoml/BentoML/issues/1339
